dev/release/setup-centos7.sh

```diff
@@ -0,0 +1,90 @@
+
+#!/bin/sh
```

Comment:
We can assume `/bin/bash` not `/bin/sh` on CentOS 7:

```suggestion
#!/bin/bash
```